src/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -80,40 +91,18 @@ export class NavigationTabs extends React.Component<Props> {
 
     const route = tab => `/search2${tab}?term=${term}`
 
-    const artistAggregationCount = get(
-      this.aggregationFor("artist"),
-      agg => agg.count,
-      0
-    )
-    const collectionAggregationCount = get(
-      this.aggregationFor("marketing_collection"),
-      agg => agg.count,
-      0
-    )
-    const galleryAggregationCount = get(
-      this.aggregationFor("PartnerGallery"),
-      agg => agg.count,
-      0
-    )
-    const showAggregationCount = get(
-      this.aggregationFor("partner_show"),
-      agg => agg.count,
-      0
-    )
-    const categoriesAggregationCount = get(
-      this.aggregationFor("gene"),
-      agg => agg.count,
-      0
-    )
-    const articlesAggregationCount = get(
-      this.aggregationFor("article"),
-      agg => agg.count,
-      0
-    )
-    const auctionsAggregationCount = get(
-      this.aggregationFor("sale"),
-      agg => agg.count,
-      0
+    const tabCountMap = Object.entries(TAB_NAME_MAP).reduce(
```

Comment:
This is all a refactor to DRY up the tab/count rendering, and also this now doesn't render tabs where the counts are 0l